executor/executor.go

```diff
@@ -1,32 +1,88 @@
 package executor
 
 import (
+	"bufio"
+	"fmt"
+	"log"
+	"os/exec"
+	"sync"
+
 	"github.com/screwdriver-cd/sd-cmd/screwdriver/api"
 	"github.com/screwdriver-cd/sd-cmd/util"
 )
 
 // Executor is a Executor endpoint
 type Executor interface {
-	Run() ([]byte, error)
+	Run() error
 }
 
 // New returns each format type of Executor
 func New(sdAPI api.API, args []string) (Executor, error) {
-	ns, cmd, ver, itr, err := util.SplitCmdWithSearch(args)
+	ns, name, ver, itr, err := util.SplitCmdWithSearch(args)
 	if err != nil {
 		return nil, err
 	}
-	sdCmd, err := sdAPI.GetCommand(ns, cmd, ver)
+
+	if itr > 2 {
```

Comment:
Does this 2 means the least number of args? Defining const variable and referring it might be prefer.

Reply:
https://github.com/screwdriver-cd/sd-cmd/pull/5/files#diff-a7c6e4fce2f5527fa5a7685c5e51d27fR52

I remember you use ```2``` here too. Could you use same const value for these ```2```? I'm sorry if it's my misunderstanding.

Reply:
sd-cmd.go's 2 and executor.go's 2 has different meaning. 
I will define each value as const.